README.md

```diff
@@ -130,6 +130,7 @@ The stack has a `SecretsBucket` parameter which will allow your build agents to
 
 The secrets bucket can contain the following files:
 
+* `/env` - An optional bash script to use as a global [agent environment hook](https://buildkite.com/docs/agent/hooks)
```

Comment:
Are these docs actually correct? Looks like the contents of these files are just eval'd, which is slightly terrible. Thoughts @sj26?